dbms/src/Storages/KVStore/MultiRaft/RegionSerde.cpp

```diff
@@ -47,20 +44,44 @@ size_t Region::writePersistExtension(
     return total_size;
 }
 
-std::tuple<size_t, UInt64> Region::serialize(WriteBuffer & buf) const
+std::tuple<size_t, UInt64> Region::serialize(WriteBuffer & buf, const RegionSerdeOpts & region_serde_opts) const
 {
+    auto expected_total = 0;
+    std::optional<std::string> maybe_large_txn_seri = std::nullopt;
+    if unlikely (region_serde_opts.large_txn_enabled)
+    {
+        maybe_large_txn_seri = data.serializeLargeTxnMeta();
+        if unlikely (maybe_large_txn_seri.has_value())
+        {
+            expected_total += 1;
+        }
```

Comment:
why unlikely?

Reply:
if large_txn_enabled = true, is `maybe_large_txn_seri.has_value()` nearly hardly happen?

Reply:
Yes, even if `large_txn_enabled = true` is true, it seldomly happens.
Currently, it never happens.

Reply:
ok